.github/workflows/check-generated-code.yml

```diff
@@ -0,0 +1,36 @@
+name: Check generated code
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "**/*.md"
+      - "LICENSE"
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "**/*.md"
+      - "LICENSE"
+
+jobs:
+  check-generated-code:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - name: Install Go
+        uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5
+        with:
+          go-version: 1.24.x
+          cache: true
+      - name: Check generated code
+        run: |
```

Comment:
We do this already, I wonder how this wasn't catched https://github.com/corazawaf/coraza/blob/main/magefile.go#L67